core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -1921,6 +1921,7 @@ protected SqlSelect createSourceSelectForUpdate(SqlUpdate call) {
     final SqlNodeList selectList = new SqlNodeList(SqlParserPos.ZERO);
     selectList.add(SqlIdentifier.star(SqlParserPos.ZERO));
     int ordinal = 0;
+    SqlNodeList targetCols = call.getTargetColumnList();
```

Comment:
Is this variable being used somewhere?

Reply:
A legacy of an earlier tactic. Will remove now.